DeeployTest/generateNetwork.py

```diff
@@ -75,6 +89,24 @@
 
     tensors = graph.tensors()
 
+    # build name→type and name→offset maps
```

Comment:
There are some weird characters in the comment.

Reply:
I fixed it in [6b99a58](https://github.com/pulp-platform/Deeploy/pull/68/commits/6b99a58c696932029a2400faab90227f1a98640e)